modules/questions.py

```diff
@@ -809,7 +807,7 @@ def parse_gdoc_link(text: str) -> Optional[str]:
     Returns `None` if message doesn't contain GDoc link.
     """
     match = re.search(
-        r"https://docs\.google\.com/document/d/.+(?=\b)", text, re.I | re.S
+        r"https://docs\.google\.com/document/d/[^\s\b]+", text, re.I | re.S
```

Comment:
`[^\s\b]+` matches a comma, which doesn't sound desired...

`[\w-]+` is more accurate (or `[a-zA-Z0-9-_]+` if we want to use specification exactly https://developers.google.com/docs/api/how-tos/overview#document_id )

Reply:
thanks, I moved that fix to the other PR and changed to `\w_-`